src/lib/viewers/doc/DocFindBar.scss

```diff
@@ -122,19 +121,15 @@ $blue: #00f !default;
     }
 }
 
-.textLayer {
-    opacity: 1;
+.bp-doc {
+    .textLayer {
+        .highlight {
```

Comment:
Does this affect embedded PDF annotations, as well?

Reply:
Those seem to live on their own layer `annotationLayer`

Reply:
I played around with a few documents with annotations (drawings & links) and things still seemed to be working well